src/huak/lib.rs

```diff
@@ -1,2 +1,53 @@
 pub mod errors;
 pub mod pyproject;
+
+use std::fs;
+use std::path::{Path, PathBuf};
+
+pub fn dir_copy<U: AsRef<Path>, V: AsRef<Path>>(from: U, to: V) -> Result<(), std::io::Error> {
+    let mut stack = Vec::new();
+    stack.push(PathBuf::from(from.as_ref()));
+
+    let output_root = PathBuf::from(to.as_ref());
+    let input_root = PathBuf::from(from.as_ref()).components().count();
+
+    while let Some(working_path) = stack.pop() {
+        println!("process: {:?}", &working_path);
```

Comment:
i *think* we want

```suggestion
        dbg!("process: {:?}", &working_path);
```

So that it's not printing this with usage.